plugins/netskope/plugin.spec.yaml

```
@@ -0,0 +1,413 @@
+        type: string
+        required: false
+      modify_time:
+        title: Modify Time
+        description: Last URL list modification time
+        type: string
+        required: false
+      pending:
+        title: Pending
+        description: URL list status of pending (1) and applied (0)
+        type: integer
+        required: false
+  replace_url_list_by_id:
+    title: Replace a URL List Configuration by ID
+    description: Completely replace the configuration of the given URL list ID
+    input:
+      id:
+        title: ID
+        description: ID of the URL list
+        type: integer
+        required: true
+        example: 1
+      name:
+        title: Name
+        description: Name of replaced URL list
+        type: string
+        required: true
+        example: ExampleName
+      urls:
+        title: URLs
+        description: List that contains URLs
+        type: "[]string"
+        required: true
+        example: '["https://example.com", "https://example.com"]'
+      type:
+        title: Type
+        description: Type of an entered URLs list
+        type: string
+        enum:
+          - ""
+          - exact
+          - regex
+        required: true
+        example: exact
+    output:
+      id:
+        title: ID
+        description: URL list identifier
+        type: integer
+        required: false
+      name:
+        title: Name
+        description: URL list name
+        type: string
+        required: false
+      data:
+        title: Data
+        description: Response data contains urls and type
+        type: object
+        required: false
+      modify_type:
+        title: Modify Type
+        description: Shows last modification type. Expected values are Created, Edited, Deleted
+        type: string
+        required: false
+      modify_by:
+        title: Modify by
+        description: Contains name that describes who modified the URL list
+        type: string
+        required: false
+      modify_time:
+        title: Modify Time
+        description: Last URL list modification time
+        type: string
+        required: false
+      pending:
+        title: Pending
+        description: URL list status of pending (1) and applied (0)
+        type: integer
+        required: false
+  delete_url_list_by_id:
+    title: Delete a URL List by ID
+    description: Mark a URL list as pending deletion
+    input:
+      id:
+        title: ID
+        description: ID of the URL list
+        type: integer
+        required: true
+        example: 1
+    output:
+      id:
+        title: ID
+        description: URL list identifier
+        type: integer
+        required: false
+      name:
+        title: Name
+        description: URL list name
+        type: string
+        required: false
+      data:
+        title: Data
+        description: Response data contains urls and type
+        type: object
```

```suggestion
        description: Data containing URLs and type
        type: object
```
Can you also type this more specifically?